tfx/components/statistics_gen/executor.py

```diff
@@ -87,6 +87,24 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     """
     self._log_startup(input_dict, output_dict, exec_properties)
 
+    # Load and deserialize exclude splits from execution properties.
+    exclude_splits = json_utils.loads(
+        exec_properties.get(EXCLUDE_SPLITS_KEY, 'null')) or []
+    if not isinstance(exclude_splits, list):
+      raise ValueError('exclude_splits in execution properties needs to be a '
+                       'list. Got %s instead.' % type(exclude_splits))
+
+    examples = artifact_utils.get_single_instance(input_dict[EXAMPLES_KEY])
+    statistics_artifact = artifact_utils.get_single_instance(
+        output_dict[STATISTICS_KEY])
```

Comment:
you can update line 139 to use this stats_artifact instead of reaccess from output dict, same eleswhere